s3-lambda-bedrock/README.md

```diff
@@ -0,0 +1,56 @@
+# S3 to Lambda to Bedrock Embeddings
+
+This pattern explains how to deploy a SAM application with Amazon S3, AWS Lambda and Amazon Bedrock.
+
+This pattern is useful for LLM RAG Usecases in getting vector embeddings of the documents uploaded to S3 via S3 event notification. Once the files are placed in S3, AWS Lambda picks up the files via event notification and post the content to Amazon Bedrock embedding model and gets the vector embedding of the content as response. 
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+* [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+* [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [AWS Serverless Application Model](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) (AWS SAM) installed
+
+## Deployment Instructions
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+    ``` 
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+1. Change directory to the pattern directory:
+    ```
+    cd s3-lambda-bedrock
+    ```
+1. From the command line, use AWS SAM to deploy the AWS resources for the pattern as specified in the template.yml file:
+    ```
+    sam build
+    sam deploy --guided
+    ```
+1. During the prompts:
+    * Enter a stack name
+    * Enter the desired AWS Region
+    * Allow SAM CLI to create IAM roles with the required permissions.
```

Comment:
it asks for a content bucket, if this is not automatically created please add it to the requirements. If not create it as IaC in the template

Reply:
modified and committed